packages/common/validators/src/validators/is-age-en.md

```diff
@@ -0,0 +1,19 @@
+---
+title: is-age
```

Comment:
We plan to match the documentation’s title with the actual function name. What about changing the title to `isAge`? (And other docs accordingly)

Reply:
[fix: match documentations't title with actual function name](https://github.com/toss/slash/pull/56/commits/589cdba91ca4c8e80cc6c3c2094315d9a8239d43)
[589cdba](https://github.com/toss/slash/pull/56/commits/589cdba91ca4c8e80cc6c3c2094315d9a8239d43)